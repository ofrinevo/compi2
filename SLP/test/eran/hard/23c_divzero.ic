
class MainClass { 

	static void main (string[] args) { 
		
		int a;
		int b = 0;

		Library.println("5 / b = ");
		a = 5 / b;
		Library.printi(a);
		Library.println("");
		Library.println("Should not print this or get here.");

	}

}
