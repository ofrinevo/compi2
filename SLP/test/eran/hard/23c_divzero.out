5 / b = 
Runtime Error: Division by zero!
